frontend/js/src/utils/utils.tsx

```diff
@@ -982,7 +982,7 @@ export function getReviewEventContent(
             readonly
             onClick={() => {}}
             className="rating-stars"
-            ratingValue={eventMetadata.rating * 20} // CB stores ratings in 0 - 5 scale but the component requires 0 - 100
+            ratingValue={eventMetadata.rating} // CB stores ratings in 0 - 5 scale but the component requires 0 - 100 UPDATE: component requires 0-5 scale too
```

Comment:
This comment can just be removed, it doesn't serve any purpose now.
As this is a very minor change I will allow myself to do it to save us both some time, and merge this straight away :)

Good job !
```suggestion
            ratingValue={eventMetadata.rating}
```